docker/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM ubuntu:impish
+
+RUN apt update && apt install openjdk-11-jre-headless wget cron -y
+RUN mkdir /opt/app
+
+RUN wget https://collectors.sumologic.com/rest/download/deb/64 -O SumoCollector.deb && chmod +x SumoCollector.deb
+RUN dpkg -i SumoCollector.deb
+RUN rm SumoCollector.deb
```

Comment:
Put all these `RUN` commands in to a single one joined with `&&`. Makes the image about 100MB smaller.